modules/virt-setting-mtv-lm-feature-gates.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-enabling-cclm-for-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-setting-mtv-lm-feature-gates_{context}"]
+= Setting a live migration feature gate in the {mtv-first}
+
+Enable the {product-title} live migration feature gate in the {mtv-first} by modifying the `spec` stanza in the `forklift-controller` custom resource (CR).
```

Comment:
```suggestion
[role="_abstract"]
Enable the {product-title} live migration feature gate in the {mtv-first} by modifying the `spec` stanza in the `forklift-controller` custom resource (CR).
```
Note, for the CQAs, the abstract should explain _why_ a user would do a procedure, so you might want to explain that here, since it just mentions how.